src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -118,6 +128,38 @@
     }
   }
 
+  &.pf-m-plain {
+    --pf-c-context-selector__toggle--PaddingRight: var(--pf-c-context-selector__toggle--m-plain--PaddingRight);
+    --pf-c-context-selector__toggle--PaddingLeft: var(--pf-c-context-selector__toggle--m-plain--PaddingLeft);
+
+    &:not(.pf-m-text) {
+      justify-content: center;
+      width: auto;
+      color: var(--pf-c-context-selector__toggle--m-plain--Color);
+    }
+
+    &::before {
+      border: 0;
+    }
+
+    .pf-c-context-selector__toggle-button-icon {
+      line-height: var(--pf-c-context-selector__toggle--LineHeight);
+    }
+
+    &:hover,
+    &:active,
+    &.pf-m-active,
+    &:focus,
+    .pf-c-context-selector.pf-m-expanded > & {
+      --pf-c-context-selector__toggle--m-plain--Color: var(--pf-c-context-selector__toggle--m-plain--hover--Color);
+    }
+
+    &.pf-m-disabled,
+    &:disabled {
```

Comment:
Technically we don't support a disabled context selector, since the use case for the context selector is that it would always be enabled, so I don't think you need this block.